### PR TITLE
🔖 Release 2.2.0

### PR DESCRIPTION
**Pull request by phatblat**
Draft for now. Waiting for build issues to be resolved and #658 to be merged.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,20 +326,25 @@ See more examples in the demo application: [ViewController.swift](/Demo/Common/V
...
 OAuthSwift could be used with others frameworks
 
 You can sign [Alamofire](https://github.com/Alamofire/Alamofire) request with [OAuthSwiftAlamofire](https://github.com/OAuthSwift/OAuthSwiftAlamofire)
 
 To achieve great asynchronous code you can use one of these integration frameworks
-- [OAuthSwiftFutures](https://github.com/OAuthSwift/OAuthSwiftFutures) - [BrightFutures](https://github.com/Thomvis/BrightFutures)
-- [OAuthRxSwift](https://github.com/OAuthSwift/OAuthRxSwift) - [RxSwift](https://github.com/ReactiveX/RxSwift)
-- [OAuthReactiveSwift](https://github.com/OAuthSwift/OAuthReactiveSwift) - [ReactiveSwift](https://github.com/ReactiveCocoa/ReactiveSwift)
+
+- [OAuthSwiftFutures](https://github.com/OAuthSwift/OAuthSwiftFutures)
```
Comment (phimage) [inline README.md:342]: not sure, here, OAuthSwiftFutures and BrightFutures must be aligned

**Comment by phimage** [comment]
we skip 2.1.3 ? too many change and maybe breaking one
(and so people could keep 2.1 if they want

File: README.md (hunk 1 above)
Comment (phatblat) [reply README.md:342]: Ah, I see. These pairs of links were meant to be on the same line.

Screen Shot 2021-05-19 at 10 58 13 AM

I'll restore this.

**Comment by phatblat** [comment]
Yeah, #596 added a new feature & API so I figured this was worth a minor version bump.

@phimage Any ideas why the BitBucket and DropBox tests fail whenever run by the GH action? 

> /Users/runner/work/OAuthSwift/OAuthSwift/OAuthSwiftTests/ServicesTests.swift:79: error: -[OAuthSwiftTests.ServicesTests testBitBucket] : failed - No parameters for BitBucket. Test ignored

> /Users/runner/work/OAuthSwift/OAuthSwift/OAuthSwiftTests/ServicesTests.swift:79: error: -[OAuthSwiftTests.ServicesTests testDropBox] : failed - No parameters for Dropbox. Test ignored

Both `Services.plist` and `ServicesTest.plist` contain top-level keys for "BitBucket" and "Dropbox", but `services["BitBucket"]` appear to be returning `nil` when there is always a value when I test this locally.

**Comment by phimage** [comment]
yes,
 we could deactivate this test on CI temporary because we cannot fill consumer key and secret

the test need to fill the configuration file with some key/secret for the services, on GitHub we need to use secret but I have no right to add it (no access to project configuration)

(I loose my file with all token ! very boring to make a new one)

**Comment by phatblat** [comment]
Tried to skip actually testing these on CI, but the build stalled for 6h before timing out.

```swift
    func testBitBucket() {
        guard ProcessInfo.processInfo.environment["CI"] != "true" else { return }
        testService("BitBucket")
```

> Thu, 20 May 2021 14:07:34 GMT
> Test Case '-[OAuthSwiftTests.ServicesTests testBitBucket]' started.
> Thu, 20 May 2021 20:05:12 GMT
> Error: The operation was canceled.

**Comment by phatblat** [comment]
Going to move forward with the 2.2 release. The current CI issues can be resolved later in #662 and #660

**Pull request merged**